### DISCUSSION TITLE
Make RStudio snippets available by default in Positron

### DISCUSSION THREAD

**Discussion by olivroy** [posit-dev/positron#4354] [Q&A]
From https://github.com/posit-dev/positron/issues/3669#issuecomment-2198769250, it seems that currently, the RStudio snippets are not transferrable to Positron out of the box?

I wonder if anyone has been able to transfer their RStudio snippets to positron?

Even better, it would be great if Positron read the RStudio snippets file and converted it to VScode snippets  https://usethis.r-lib.org/reference/edit.html

**Comment by petetronic** [comment]
I believe @gadenbuie mentioned an unofficial gist that did this https://gist.github.com/gadenbuie/7fe84038a5bfd39bbcc6ef13b47a3420

... though we do not yet have a tool in Positron that would do this.

**Comment by petetronic** [comment]
Note that Positron moved built-in R snippets to ARK https://github.com/posit-dev/ark/pull/183
